subsys/disk/disk_access_spi_sdhc.c

```diff
@@ -778,6 +783,17 @@ static int sdhc_spi_init(struct device *dev)
 	data->pin = DT_SPI_DEV_CS_GPIOS_PIN(DT_INST(0, zephyr_mmc_spi_slot));
 	data->flags = DT_SPI_DEV_CS_GPIOS_FLAGS(DT_INST(0, zephyr_mmc_spi_slot));
 
+
+	/* TODO why is active low not picked up */
+	data->flags |= GPIO_ACTIVE_LOW;
```

Comment:
Could you please post you full device tree declaration for this one ? This includes your full SPI node with CS and mmc spi slot. There must be some small detail we are missing.

Reply:
`
&spi2  {
    status = "okay";

    dmas = <&dma1 4 0 0x28440 0x03
                   &dma1 3 0 0x28480 0x03>;

    dma-names = "tx", "rx";

    cs-gpios = <&gpiod 14 GPIO_ACTIVE_LOW>, <&gpiod 3 GPIO_ACTIVE_LOW>;

    oled@0 {
        reg = <0>;
        status = "okay";
        label = "OLED0";
        spi-max-frequency = <5400000>;
    };

     sdhc0: sdhc@1 {
          compatible = "zephyr,mmc-spi-slot";
          reg = <1>;
          status = "okay";
          label = "SDHC0";
          spi-max-frequency = <5400000>;
     };
};
`